cql/operation/fhir-operation-cpg/src/test/java/com/ibm/fhir/operation/cpg/CqlOperationTest.java

```diff
@@ -54,6 +60,75 @@
     protected CqlOperation getOperation() {
         return new CqlOperation();
     }
+    
+    @Test
+    public void testRequestContainsUnsupportedParameters() throws Exception {
+        Patient patient = (Patient) TestHelper.getTestResource("Patient.json");
+
+        String codesystem = "http://snomed.ct/info";
+        String encounterCode = "office-visit";
+        Coding reason = coding(codesystem, encounterCode);
+
+        Encounter encounter = Encounter.builder()
+                .reasonCode(concept(reason))
+                .status(EncounterStatus.FINISHED)
+                .clazz(reason)
+                .period(Period.builder()
+                    .start(DateTime.now())
+                    .end(DateTime.now())
+                    .build())
+                .build();
+
+        String expression = "[Encounter] e where e.status = 'finished'";
+
+        Parameter pSubject = Parameter.builder().name(fhirstring(CqlOperation.PARAM_IN_SUBJECT)).value(fhirstring("Patient/" + patient.getId())).build();

```

Comment:
you shouldn't need to use `fhirstring` any more, you can pass your java string directly